themes/src/main/resources/theme/base/admin/resources/js/services.js

```diff
@@ -387,6 +387,14 @@ module.factory('RealmSpecificLocalizationTexts', function($resource) {
     return $resource(authUrl + '/admin/realms/:id/localization/:locale', {
         id : '@realm',
         locale : '@locale'
+    }, {
+        get: {
+            isArray: false,
+                method: 'GET',
+                params: {
+                    useRealmDefaultLocaleFallback: '@useRealmDefaultLocaleFallback'
```

Comment:
Perhaps this should just be the default behavior instead of passing an explicit parameter?

Reply:
@leischt I tend to agree with @jonkoops here.  Is there any reason someone would want the old behavior?

Reply:
There is also a UI page to manage the localization texts. This page should only show the texts which are defined for a given locale and don't use the fallback feature. I'm not sure if it is better to specify the value of the parameter "useRealmDefaultLocaleFallback" each time or have a default and modify it for one of the calls.

Relevant call for the localization management UI page: https://github.com/keycloak/keycloak/blob/b34f46155c494bb3944913eb94cd6bdb845eae33/themes/src/main/resources/theme/base/admin/resources/js/controllers/realm.js#L614